packages/upswyng-server/static/manifest.json

```diff
@@ -1,8 +1,20 @@
 {
   "background_color": "#ffffff",
-  "theme_color": "#333333",
-  "name": "TODO",
-  "short_name": "TODO",
+  "theme_color": "#f15b29",
```

Comment:
uhm, this isn't [our orange](https://github.com/CodeForBoulder/upswyng/blob/master/packages/upswyng-web/src/App.styles.ts#L19). This looks [waaaaayyyyy different](https://www.google.com/search?q=%23F05A28&oq=%23F05A28&aqs=chrome..69i57j0.1257j0j7&sourceid=chrome&ie=UTF-8) 😏

Reply:
Ummm you sure?

https://github.com/CodeForBoulder/upswyng/blob/master/packages/upswyng-server/src/style/main.scss#L1

<img width="934" alt="Screen Shot 2020-01-27 at 9 36 39 PM" src="https://user-images.githubusercontent.com/5778036/73235797-222ec880-414d-11ea-88a6-7db3a3029e27.png">

Reply:
<img width="663" alt="Screen Shot 2020-01-27 at 9 37 25 PM" src="https://user-images.githubusercontent.com/5778036/73235823-3c68a680-414d-11ea-92ef-cd48f304c7a2.png">

Reply:
Is that derived from the SVG or PNG? Either way, I really was just joking :P

Reply:
It's from some Illustrator file in one of Mike's Google Drive folders. 🤷🏾‍♀️ I legit thought you had like crazy color perception or something.